libs/blocks/merch-cards/merch-cards.js

```diff
@@ -5,6 +5,7 @@ import '../merch/merch.js';
 import '../merch-card/merch-card.js';
 import { createTag, decorateLinks, getConfig, loadBlock, loadStyle } from '../../utils/utils.js';
 import { replaceText } from '../../features/placeholders.js';
+import { OVERRIDE_PATHS } from '../../features/personalization/personalization.js';
```

Comment:
Not ideal for performance: personalization.js is always loaded when merch cards are, even if nothing else is used.